### PR TITLE
Ad detail

### PR DESCRIPTION
**Pull request by ahmaziz12**

### CODE REVIEW

File: app/views/ads/show.html.erb [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div class="d-flex justify-content-center mt-3 mb-2">
      <h1>
           Ad Detail
-        </h1>
+      </h1>
```
Comment (asad-imtiaz) [inline app/views/ads/show.html.erb:5]: identation

File: app/views/ads/show.html.erb [hunk 2]
```diff
@@ -30,8 +30,8 @@
               <% else %>
                 <div class="carousel-item">
               <% end %>
-                  <%= image_tag img, class: "d-block w-100" %>
-                  </div>
+              <%= image_tag img, class: "d-block w-100" %>
```
Comment (asad-imtiaz) [inline app/views/ads/show.html.erb:33]: remove if conditions for active class

File: app/views/ads/show.html.erb (hunk 1 above)
Comment (ahmaziz12) [reply app/views/ads/show.html.erb:5]: Done

File: app/views/ads/show.html.erb (hunk 2 above)
Comment (ahmaziz12) [reply app/views/ads/show.html.erb:33]: Done

**Pull request merged**